examples/openfermionprojectq_demo.ipynb

```diff
@@ -26,7 +24,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
```

Comment:
No action item for you here but someday it would nice to figure out how to have git ignore changes like this.